scripts/copyIcons.mjs

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
```

Comment:
I'm feeling fine with this script for now. I'd like to avoid file operations in our build whenever we can, so making it easier via a vite plugin doesn't make sense to me at this time